docs/calicoctl/resources/policy.md

```diff
@@ -0,0 +1,89 @@
+
+### Sample YAML
+```
+apiVersion: v1
+kind: policy
+metadata:
+  name: policy1
+  tier: tier1
+spec:
+  order: 100
+  selector: type=='database'
+  ingress:
+  - action: deny
+    protocol: tcp
+    icmp:
+      - type: 10
+      - code: 6
+    "!protocol": ipv6
+    "!icmp":
+      - type: 19
+      - code: 255
+    source:
+      tag: footag
+      net: 10.0.0.0/16
+      selector: type=='application'
+      ports: 1234,10:20
+      "!tag": bartag
+      "!net": 10.1.0.0/16
+      "!selector": type=='database'
+      "!ports": 1050
+    destination:
+      tag: alphatag
+      net: 10.2.0.0/16
+      selector: type=='application'
+      ports: 100:200
+      "!tag": type=='bananas'
+      "!net": 10.3.0.0/16
+      "!selector": type=='apples'
+      "!ports": 1050:110
+  egress:
+  - action: allow
+    source:
+      selector: type=='application'
+```
+> Note: The above YAML spec defines almost all of possible fields for a policy specifications, with one exception - the "egress" section has been simplified for readability, despite the fact that it supports all fields that "ingress" does.
+
+#### Metadata
+| name | description  | requirements                  | schema |
+|------|--------------|-------------------------------|--------|
+| name | The name of the policy. | | string |
+| tier | The name of the parent tier. | If omitted, assumed to mean the "default" tier - which is automatically created by the calicoctl and is the last tier to be acted on. | string |
+
+
+#### PolicySpec
+| name     | description                                                          | requirements | schema |
+|----------|----------------------------------------------------------------------|--------------|--------|
+| order    | The order number which indicates the order that this policy is used. | | integer |
+| ingress  | The ingress rules belonging to this policy.                          | | List of [RuleSpecs](#rulespec) |
+| egress   | The egress rules belonging to this policy.                           | | List of [RuleSpecs](#rulespec)  |
+| selector | Selector expression.                                                 | See [selector expression documentation](http://docs.projectcalico.org/en/latest/etcd-data-model.html#tiered-security-policy) | string |
+
+#### RuleSpec
+| name        | description                                | requirements | schema |
+|-------------|--------------------------------------------|----------------|--------|
+| action      | Action to perform when matching this rule.  Can be one of: `nextTier`, `allow`, `deny` |  | string |
+| protocol    | Positive protocol match.  | Can be one of: `tcp`, `udp`, `icmp`, `icmpv6`, `sctp`, `udplite`, or an integer 1-255. | string |
+| icmpType    | Positive match on ICMP type. | Can be integer 1-255 | integer |
```

Comment:
this should be `icmp` and subtype `type` and `code` (`icmpType` , `icmpCode`, `!icmpType` and `!icmpCode` are no longer the keywords as of #93 ) 

```yaml
icmp:
    - type: 10
    - code: 6
```